### PR TITLE
Set request so that the payload/query and other attributes are accessible in the seneca action

### PR DESCRIPTION
**Pull request by mirceaalexandru**
request.raw.req contains the original request, however the request object contains parsed payload/query/authentication and other information.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ internals.request = function (seneca) {
     return (request) => {
 
         return seneca.delegate({
-            req$: request.raw.req,
```
Comment (geek) [inline lib/index.js]: thoughts on dropping `res$`?  Since its accessible from `res$` after this change.

Comment (mirceaalexandru) [inline lib/index.js]: Yes, you are right, I will remove it.

**Comment by AdrieanKhisbe** [comment]
@mirceaalexandru  interesting :smiley:

**Comment by mirceaalexandru** [comment]
All changes from this PR are also in https://github.com/hapijs/chairo/pull/32, so I will close this.

**Pull request closed without merging**